x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/flyout_components/add_to_rules_table/index.test.tsx

```diff
@@ -11,12 +11,13 @@ import { mountWithIntl } from '@kbn/test-jest-helpers';
 import { ExceptionsAddToRulesTable } from '.';
 import { TestProviders } from '../../../../../common/mock';
 import { useFindRulesInMemory } from '../../../../rule_management_ui/components/rules_table/rules_table/use_find_rules_in_memory';
-import { getRulesSchemaMock } from '../../../../../../common/detection_engine/rule_schema/mocks';
-import type { Rule } from '../../../../rule_management/logic/types';
+// import { getRulesSchemaMock } from '../../../../../../common/detection_engine/rule_schema/mocks';
+// import type { Rule } from '../../../../rule_management/logic/types';
 
 jest.mock(
   '../../../../rule_management_ui/components/rules_table/rules_table/use_find_rules_in_memory'
 );
+// TODO convert this test to RTL
```

Comment:
disabling Test for now as anyway I need to convert this test to use react-testing-library